src/Plugins/Nop.Plugin.ExternalAuth.Facebook/Migrations/UpgradeTo460/LocalizationMigration.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using FluentMigrator;
+using Nop.Core.Infrastructure;
+using Nop.Data;
+using Nop.Data.Migrations;
+using Nop.Services.Common;
+using Nop.Services.Localization;
+
+namespace Nop.Plugin.ExternalAuth.Facebook.Migrations.UpgradeTo460
```

Comment:
No need for a separate folder `UpgradeTo460`, the `Migrations` is enough for the plugin.